tests/itest_specs/basic_round_trip/short_call.cairo

```diff
@@ -575,20 +572,20 @@ namespace ShortCallRoundTrip {
             contract_address=amm_addr,
             lptoken_address=lpt_call_addr
         );
-        assert call_pool_unlocked_capital_3 = 6917838435345032594;
+        assert call_pool_unlocked_capital_3.low = 3000134184201922856;
 
         let (put_pool_unlocked_capital_3) = ILiquidityPool.get_unlocked_capital(
             contract_address=amm_addr,
             lptoken_address=lpt_put_addr
         );
-        assert put_pool_unlocked_capital_3 = 11529215046068469760000;
+        assert put_pool_unlocked_capital_3.low = 5000000000;
 
         // // Test balance of option tokens in the account after the option was bought and after withdraw
         let (bal_opt_short_tokens_3: Uint256) = IOptionToken.balanceOf(
             contract_address=opt_short_call_addr,
             account=admin_addr
         );
-        assert bal_opt_short_tokens_3.low = 500000000000000000;
+        assert bal_opt_short_tokens_3.low = 783159565502899114;
```

Comment:
potential BUG!!!

half of the position was closed... by the
```
        let (premia_2: Math64x61_) = IAMM.trade_close(
            contract_address=amm_addr,
            option_type=0,
            strike_price=strike_price,
            maturity=expiry,
            option_side=1,
            option_size=half,
            quote_token_address=myusd_addr,
            base_token_address=myeth_addr
        );
```
which means only half of the option tokens should be remaining... how is it possible it is not the case?